req.sh

```diff
@@ -0,0 +1,11 @@
+ curl -X POST -H "Content-Type: application/json" -d '{
```

Comment:
I don't think we need this file. The documentation APIs can be shared in a folder, as part of the next documentation MR (in l2 format, of course)

Reply:
Right